src/main.ts

```diff
@@ -20,16 +20,17 @@ async function bootstrap() {
 
   app.use('/interaction', urlencoded({ extended: false }));
 
-
   const config = new DocumentBuilder()
     .setTitle('OIDC Protocol setup')
     .setDescription('This is the project to implement OIDC protocol in nestjs')
     .setVersion('1.0')
     .build();
   const document = SwaggerModule.createDocument(app, config);
   SwaggerModule.setup('api', app, document);
-  // saving the swagger file 
-  fs.writeFileSync('./swagger-spec.json', JSON.stringify(document, null, 2), { encoding: 'utf8' });
+  // saving the swagger file
+  fs.writeFileSync('./swagger-spec.json', JSON.stringify(document, null, 2), {
```

Comment:
Please use await while using read file sync if its important. Or we can just use readFile.